### PR TITLE
Add Tranche Drawdown Report

### PR DESCRIPTION
**Pull request by sidane**
- Cfe User only report
- Includes details of premium schedules with tranche drawdowns
- Restricted to premium schedules for guaranteed loans
- Restricted to only the latest premium schedule for loans

### CODE REVIEW

File: app/controllers/tranche_drawdown_reports_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class TrancheDrawdownReportsController < ApplicationController
+  before_filter :verify_create_permission
```
Comment (nickcharlton) [inline app/controllers/tranche_drawdown_reports_controller.rb]: I believe `before_filter` is deprecated in favour of `before_action`: http://guides.rubyonrails.org/action_controller_overview.html#filters

File: spec/reports/tranche_drawdown_report_spec.rb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          loan: guaranteed_loan,
+          repayment_duration: 15,
+          initial_capital_repayment_holiday: 0,
+          initial_draw_amount: Money.new(12_000_00),
+          second_draw_amount: Money.new(3_000_00),
+          second_draw_months: 9,
+          seq: 0
+        )
+
+        _latest_premium_schedule_with_tranche_drawdowns = create(
```
Comment (nickcharlton) [inline spec/reports/tranche_drawdown_report_spec.rb]: I like the use of the `_` to give them a name, as this is so damn complex!

**Comment by nickcharlton** [comment]
One comment, otherwise looks good!

File: app/controllers/tranche_drawdown_reports_controller.rb (hunk 1 above)
Comment (sidane) [inline app/controllers/tranche_drawdown_reports_controller.rb]: Good spot, will update that and make a card in Trello to update the other controllers in a separate PR.

**Comment by sidane** [comment]
Thanks @nickcharlton

**Pull request merged**